http_check/test_http_check.py

```diff
@@ -239,7 +239,7 @@ def test_check(self):
         """
         self.run_check(CONFIG)
         # Overrides self.service_checks attribute when values are available\
-        self.service_checks = self.wait_for_async('get_service_checks', 'service_checks', 5)
+        self.service_checks = self.wait_for_async('get_service_checks', 'service_checks', 6)
```

Comment:
Because I think 6 is the number of service checks we indeed do receive. It's not a time value, but a hit count value.